governance.md

```diff
@@ -1,34 +1,61 @@
-* prioritize issues and manage their processing
-* proactively fix bugs
-* perform maintenance tasks for their projects
+With great power comes great responsibility. Code owners not only review and approve PRs but also truly care about their projects.
 
-## Become a maintainer
+Every code owner is expected to:
 
-To become a maintainer, contact one of the existing maintainers of the given project and agree to fulfill the requirements listed above.
-The existing maintainer can also propose a new member by creating a pull request adding the candidate to the `CODEOWNERS` file in the project's repository. The change requires the two-third majority of all maintainers' votes.
+* Contribute high-quality code and content
+* Communicate and collaborate with other code owners to improve the ownership process
+* Perform a thorough review of incoming PRs and make sure they follow the [contributing rules](CONTRIBUTING.md)
+* Approve only those PRs in which the contributor made the requested improvements
+* Check if the related CI job has completed successfully before approving the PR
+* Make sure that the PR approval flow runs smoothly
+* Prioritize issues and manage their processing
+* Proactively fix bugs
+* Perform maintenance tasks for their projects
 
-## Remove a maintainer
+### Add or remove a code owner
 
-The maintainer's role requires commitment and ongoing contribution to a project. If a maintainer is no longer able to fulfill these requirements or does not contribute to the project for half a year, remove them from the list. The change requires the two-third majority of all maintainers' votes.
+To suggest a change in the ownership of a given repository part, create a PR with the required changes in the `CODEOWNERS` file in the project's repository. The required number of code owners needs to approve the PR for the changes to take place. Read [here](https://github.com/kyma-project/community/blob/master/guidelines/internal-guidelines/repository-template/template/CODEOWNERS) how to set up and modify owners of the given repository folders and files.
+
+## The decisions-making process
+
+In general, the relevant Special Interest Groups (SIGs) and Working Groups (WGs) make the decisions that affect the project, including its structure, functionalities, components, or work of the project teams. However, the organizational decisions and those that relate to the product strategy belong to the Kyma Council.
+
+SIGs and WGs follow the "lazy consensus" decision-making process which assumes that:
+
+* All SIG/WG members have an equal voice in the decision-making process.
+* Silence is consent. By default, lack of objections to a proposed decision means a silent approval.
+* Any objections are good opportunities for healthy and constructive discussions.
+
+> **NOTE:** The described approach only concerns the decisions taken during SIG and WG meetings. It does not affect any Kyma decisions taken during daily team activities.
```

Comment:
since SIGs and WG do work all over the week not just during meetings, I would suggest to remove `meetings` from this sentence. Maybe reword around : only concerns the SIG and WG related decisions. OR: only concerns decisions being made as part of SIGs and WGs activities.

Reply:
`concerns the decisions made by SIGs and WGs.`